sqlflow/magic.py

```diff
@@ -2,6 +2,7 @@
 import logging
 
 from IPython.core.magic import Magics, magics_class, cell_magic, line_magic
+from IPython.display import display_javascript
```

Comment:
Hard to imagine that the import of a package named javascript is for highlight SQL syntax. We need a comment here.

Reply:
Done.